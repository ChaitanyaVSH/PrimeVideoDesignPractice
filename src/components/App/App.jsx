import React from "react";
import VideoCard from "../VideoCard/VideoCard";
import styles from "./App.module.css";

const App = () => {
    return(
        <div className={styles.App}>
            <h1>Welcome to My world</h1>
            <VideoCard/>
        </div>
    );
}

export default App;
