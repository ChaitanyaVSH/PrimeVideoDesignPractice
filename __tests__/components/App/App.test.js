import React from "react";
import { shallow } from 'enzyme';
import App from "../../../src/components/App/App";

describe("VideoCard Component is rendering",()=>{
    it("should render without any errors",() => {
        const wrapper = shallow(<App/>)
        expect(wrapper.type()).toBe('div')
    });

    it("should maintain the older snapshot",() => {
        const wrapper = shallow(<App/>)
        expect(wrapper).toMatchSnapshot();
    })
});
