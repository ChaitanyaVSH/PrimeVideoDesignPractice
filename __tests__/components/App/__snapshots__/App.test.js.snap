// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App Component is rendering should maintain the older snapshot 1`] = `
<div
  className="App"
>
  <h1>
    Welcome to Prime Video
  </h1>
  <VideoCard />
</div>
`;
